src/modules/client/components/ClientProfileCard.tsx

```diff
@@ -269,6 +269,8 @@ const ClientCardImage = ({
 
   if (!client) return <ClientCardImageElement />;
 
+  if (!client.access.canViewClientPhoto) return;
```

Comment:
We should do a design pass at some point, it seems like the edit button could flow to the right maybe. There are some installations that may never want to use photos, and now those installations can just turn off this permission, which is great. 👍

Reply:
Although, it looks like the loading box still appears momentarily, and then disappears. Lets move this line up above 270 so that doesn't happen

Reply:
Hm, moving it above line 270 means the client could be undefined. But I've solved it one level up by checking the root permissions, here: https://github.com/greenriver/hmis-frontend/pull/726/files#diff-f2bf835ebbcb6e05582acd008cd571838fa17cdafc8a48b58a85b24ae41541c1R386-R400